test/helpers/api-unit.helper.js

```diff
@@ -0,0 +1,46 @@
+// @TODO: remove when lodash can be upgraded
+import defaults from 'lodash.defaultsdeep';
+import { model as User } from '../../website/src/models/user'
+import { model as Group } from '../../website/src/models/group'
+import i18n from '../../common/script/src/i18n';
+require('coffee-script');
+i18n.translations = require('../../website/src/libs/i18n.js').translations;
+
+afterEach(() => {
+  sandbox.restore();
+});
+
+export function generateUser(options={}) {
+  return new User(options)._doc;
```

Comment:
I thought I tried it and it didn't work like I expected. But I just tried it again and it does, so we'll go with that. 

```
¯\_(ツ)_/¯
```